client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
```

Comment:
It looks like when selecting a new date, the calendar doesn't remove the old/previous date. (I.e. both dates are highlighted)

![image](https://user-images.githubusercontent.com/24628243/156708489-f5b6ce8e-91b8-4448-8787-90a826f324b9.png)

Reply:
Changed today's date style to underline.

Reply:
Although it is underlined, this should be more visible to the end user. It even took me a while to find the underline for the current date 😅 

![image](https://user-images.githubusercontent.com/24628243/157380318-ccefd020-70b0-41b7-b632-5db50a361bb7.png)